eval.py

```diff
@@ -4,77 +4,195 @@
 # right wandb run
 import argparse
 import json
+import logging
 from pathlib import Path
-from typing import List
+from typing import List, cast
 
 import torch
 from tqdm import tqdm
+from wandb.sdk.wandb_run import Run
 
-from presto.dataops import BANDS_GROUPS_IDX
 from presto.eval import (
     AlgaeBloomsEval,
     CropHarvestEval,
     EuroSatEval,
-    EvalDataset,
+    EvalTask,
     FuelMoistureEval,
     TreeSatEval,
 )
 from presto.presto import Presto
-from presto.utils import config_dir, default_model_path, device, seed_everything
+from presto.utils import (
+    DEFAULT_SEED,
+    config_dir,
+    default_model_path,
+    device,
+    initialize_logging,
+    seed_everything,
+    timestamp_dirname,
+    update_data_dir,
+)
 
 seed_everything()
+logger = logging.getLogger("__main__")
 
 argparser = argparse.ArgumentParser()
 argparser.add_argument("--path_to_state_dict", type=str, default="")
 argparser.add_argument("--path_to_config", type=str, default="")
+argparser.add_argument(
+    "--data_dir",
+    type=str,
+    default="",
+    help="Data is stored in <data_dir>/data. "
+    "Leave empty to use the directory you are running this file from.",
+)
+argparser.add_argument(
+    "--output_dir",
+    type=str,
+    default="",
+    help="Output is stored in <data_dir>/output. "
+    "Leave empty to use the directory you are running this file from.",
+)
 argparser.add_argument("--fully_supervised", dest="fully_supervised", action="store_true")
+argparser.add_argument("--wandb", dest="wandb", action="store_true")
```

Comment:
Could you add an argument `argparser.add_argument("--nb_seeds", type=int, default=1)` so it's actually also still possible to run the eval script with just 1 seed? 
Or do we not want to support that?
Then below: `seeds = list(range(0, DEFAULT_SEED*nb_seeds, DEFAULT_SEED) if nb_seeds > 1 else [DEFAULT_SEED]`

Reply:
added an `eval_seeds` arguments